Editor/EdgegapWindowV2.cs

```diff
@@ -765,22 +766,34 @@ private void onApiTokenInputFocusOut(FocusOutEvent evt)
         {
             _apiTokenInput.isPasswordField = true;
 
-            _isApiTokenVerified = false;
-            _postAuthContainer.SetEnabled(false);
-            closeDisableGroups();
-
-            // Toggle "Verify" btn on 1+ char entered
-            if (_apiToken.Length > 0)
+            if (!_isVerifyingToken)
             {
-                onApiTokenVerifyBtnClick();
+                _isApiTokenVerified = false;
+                _postAuthContainer.SetEnabled(false);
+                closeDisableGroups();
+
+                // Toggle "Verify" btn on 1+ char entered
+                if (_apiToken.Length > 0)
```

Comment:
could we move this condition above to line 769 to avoid partial failure where we disable UI but don't launch verification?:
```cs
if (!_isVerifyingToken && _apiToken.Length > 0)
{
  _isVerifyingToken = true;
```

afterwards, `_isVerifyingToken = true;` should be the first thing we do inside the if statement body, before anything else